docs/flow/config-language/expressions/referencing_outputs.md

```diff
@@ -1,8 +1,69 @@
 ---
 aliases:
-- /docs/agent/latest/flow/configuration-language/expressions/referencing-outputs
-title: Referencing component exports
+- /docs/agent/latest/flow/configuration-language/expressions/referencing-exports
+title: Referencing component Exports
 weight: 200
 ---
 
-# Types and Values
+# Referencing component Exports
+Referencing component Exports is what enables River to dynamically configure
```

Comment:
nit: does exports need to be capitalized?

Reply:
I'm not sure, but we when decide on something it should be enforced throughout our documentation.